### DISCUSSION TITLE
Does vercel provide SSL for custom domain?

### DISCUSSION THREAD

**Discussion by Dan-Veinberg** [vercel/community#4154] [Feature Question] [answered]
### Question

If I host a static website and then  I want to create a custom domain name for it. Does vercel provide ssl certificate automatically?

**Comment by vickkhera** [comment, marked as answer]
Yes.

**Reply by amyegan** [reply]
For anyone looking for more info about automatic SSL, it can be found in the [HTTPS/SSL](https://vercel.com/docs/security/encryption#) docs. 
> Out of the box, every Deployment on Vercel is served over an HTTPS connection. The [SSL](https://en.wikipedia.org/wiki/Transport_Layer_Security) certificates for these unique URLs are automatically generated free of charge.

If you have problems with automatic SSL, some troubleshooting steps can be found here: https://vercel.com/guides/domain-not-generating-ssl-certificate
